1.10/deploying-services/gpu.md

```diff
@@ -0,0 +1,169 @@
+---
+post_title: Enabling GPU Support
+feature_maturity: preview
+menu_order: 110
+---
+
+DC/OS supports allocating GPUs (Graphics Processing Units) to your long-running DC/OS services. Adding GPUs to your service can dramatically accelerate big data workloads. 
+
+# Enabling GPUs
+GPUs must be enabled during DC/OS installation. Follow the instructions to enable GPUs based on your DC/OS deployment method.
+
+## Enabling GPUs for a Custom DC/OS Installation
+
+1.  Install DC/OS using the [custom installation instructions](/docs/1.10/installing/custom/advanced/) with the `enable-gpu-isolation: 'true'` configuration parameter specified.
```

Comment:
k, i won't mention that here, since the focus of this doc is enabling GPUs. i will include in the config parms.